src/Propel/Generator/Manager/AbstractManager.php

```diff
@@ -58,13 +58,19 @@ abstract class AbstractManager
 
     /**
      * The XSD schema file to use for validation.
-     * @var string
+     *
+     * @deprecated Not in use and not working due to missing class.
```

Comment:
What classes are missing?

Reply:
The property itself is unknown class

Reply:
Is it something that existed in an earlier version of a dep?

Reply:
Must have been a long time ago, for now this is sure dead and not working code.
I just didn't want to remove too much now, so just making it clear and visible to be tech debt.